### PR TITLE
[DE] Add `Beatmap/Category`

### PR DESCRIPTION
**Pull request by Pasi4K5**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by FamousPig — CHANGES_REQUESTED** [review]
Hey  :)

Also make sure to check CI errors (https://github.com/ppy/osu-wiki/runs/7654430045?check_suite_focus=true). Not sure how visible those are to you.

File: wiki/Beatmap/Category/de.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### Qualifiziert
+
+"Qualifiziert" ist die letzte Stufe des [Beatmap-Ranking-Verfahrens](/wiki/Beatmap_ranking_procedure). Diese Beatmaps wurden von mindestens zwei [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators), die zuvor sicherstellten, dass die Beatmap den [Ranking-Kriterien](/wiki/Ranking_Criteria) entspricht, nominiert. Qualifizierte Beatmaps werden in eine [Ranking-Warteschlange](/wiki/Beatmap_ranking_procedure/Ranking_queue) eingereiht und letztendlich, wenn sie nicht [disqualifiziert](/wiki/Beatmap_ranking_procedure#nomination-resets) werden, [gerankt](#ranked). Dieser Zeitraum dient als eine "letzte Verteidigungslinie" und schafft die Möglichkeit, bisher unbemerkte Fehler und Mängel zu beheben.
+
+Qualifizierte Beatmaps haben Ranglisten, vergeben aber keine [Performance-Punkte](/wiki/Performance_points). Weiters werden alle Scores gelöscht, sobald die Beatmap in eine andere Kategorie verschoben wird. Im [Songauswahl-Bildschirm](/wiki/Client/Interface#song-select) werden qualifizierte Beatmaps mit einem Häkchen-Symbol (![](/wiki/shared/status/qualified.png)) markiert.
```
[the remark this answers is not in the document]
Comment (Pasi4K5) [reply wiki/Beatmap/Category/de.md]: I just changed it as suggested. `Des Weiteren` is definitely correct and I don't really care which one is used, so let's just use `Des Weiteren`.

File: wiki/Beatmap/Category/de.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Loved
+
+*Für die Geschichte der Loved-Kategorie, siehe: [Geschichte von Loved](/wiki/Beatmap/History_of_Loved)*
+
+Loved Beatmaps entsprechen nicht zwingend den [Ranking-Kriterien](/wiki/Ranking_Criteria), werden aber von der Community *geliebt* und werden deshalb, basierend auf Abstimmungen der Community, im Rahmen von [Project Loved](/wiki/Community/Project_Loved) regelmäßig hinzugefügt. Loved Beatmaps verfügen über Ranglisten, vergeben aber keine [Performance-Punkte](/wiki/Performance_points). Weiters werden alle Scores gelöscht, sobald die Beatmap in eine andere Kategorie verschoben wird (dies geschieht für gewöhnlich nur auf Anfrage des Erstellers). Im [Songauswahl-Bildschirm](/wiki/Client/Interface#songauswahl) werden Loved Beatmaps mit einem Herz-Symbol (![](/wiki/shared/status/loved.png)) markiert.
```
Comment (FamousPig) [inline wiki/Beatmap/Category/de.md:33]:
```suggestion
Loved Beatmaps entsprechen nicht zwingend den [Ranking-Kriterien](/wiki/Ranking_Criteria), werden aber von der Community *geliebt* und werden deshalb, basierend auf Abstimmungen der Community, im Rahmen von [Project Loved](/wiki/Community/Project_Loved) regelmäßig hinzugefügt. Loved Beatmaps verfügen über Ranglisten, vergeben aber keine [Performance-Punkte](/wiki/Performance_points). Des Weiteren werden alle Scores gelöscht, sobald die Beatmap in eine andere Kategorie verschoben wird (dies geschieht für gewöhnlich nur auf Anfrage des Erstellers). Im [Songauswahl-Bildschirm](/wiki/Client/Interface#songauswahl) werden Loved Beatmaps mit einem Herz-Symbol (![](/wiki/shared/status/loved.png)) markiert.
```

As before.

File: wiki/Beatmap/Category/de.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Zu Beginn hatten Approved Beatmaps keinen Einfluss auf das Score-Ranking eines Spielers. Dies änderte sich jedoch kurz nach dem Wechsel von Score-Ranking zu [Performance-Punkten](/wiki/Performance_points). Nach dieser Änderung wurde die Approved-Kategorie ausschließlich für Beatmaps mit mehr als 6 Minuten [Drain-Zeit](/wiki/Gameplay/Drain_time) verwendet.
+
+Im [Songauswahl-Bildschirm](/wiki/Client/Interface#songauswahl) werden Approved Beatmaps mit einem Häkchen-Symbol (![](/wiki/shared/status/approved.png)) markiert.
+
+### Einführung der Qualifiziert-Kategorie
+
+Die [Qualifiziert](#qualifiziert)-Kategorie wurde im August 2014 während der [Umstrukturierung des BATs](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) eingeführt, um kürzlichn gerankte Beatmaps eine größere Bühne zu geben. Wenn eine qualifizierte Beatmap disqualifiziert und zurück in die Pending-Kategorie verschoben werden musste, würde es die Statistiken der Spieler nicht beeinflussen, da die Kategorie von Anfang an für das Vergeben von temporären Scores bestimmt war. Dadurch wurden richtige Erwartungen gesetzt und das negative Feedback der Community gegenüber Unranks wurde drastisch reduziert.
```
Comment (FamousPig) [inline wiki/Beatmap/Category/de.md:47]:
```suggestion
Die [Qualifiziert](#qualifiziert)-Kategorie wurde im August 2014 während der [Umstrukturierung des BATs](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) eingeführt, um kürzlich gerankten Beatmaps eine größere Bühne zu geben. Wenn eine qualifizierte Beatmap disqualifiziert und zurück in die Pending-Kategorie verschoben werden musste, würde es die Statistiken der Spieler nicht beeinflussen, da die Kategorie von Anfang an für das Vergeben von temporären Scores bestimmt war. Dadurch wurden richtige Erwartungen gesetzt und das negative Feedback der Community gegenüber Unranks wurde drastisch reduziert.
```
\*Clears throat\*

**Review by FamousPig — CHANGES_REQUESTED** [review]
Everything else looks good to me.

**Comment by The-Last-Cookie** [comment]
Also hello from me!
> it may be safer to try and stick to "standard german"
For future reference: I believe it is quite clear that the "standard german rule" as described in the [ASC](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing) excludes any other German variant. So, I fully agree with you to use `Des Weiteren` in this case.
I would like to review the article tomorrow if this is okay because I don't have time tonight.

**Comment by Pasi4K5** [comment]
> Also hello from me!
> 
> > it may be safer to try and stick to "standard german"
> 
> For future reference: I believe it is quite clear that the "standard german rule" as described in the [ASC](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing) excludes any other German variant. So, I fully agree with you to use `Des Weiteren` in this case.

Hey, so I did a little bit of research, and here's what I found out:

The [ASC](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing) only states that Standard German must be used, it doesn't rule out Austrian German (or Swiss German for that matter). The word "Standard German" is basically a collective term for the different national varieties of German (German Standard German, Austrian Standard German and Swiss Standard German). Those varieties are mostly identical to each other, apart from a few differences in vocabulary and pronunciation, and are not to be confused with dialects, which differ significantly from all of those Standard German varieties.
In other words, Austrian Standard German (which includes words like `weiters`) is no less Standard German than German Standard German.

You can read more on that topic on Wikipedia if you want: [English article](https://en.wikipedia.org/wiki/Standard_German#Pluricentricity), [German article](https://en.wikipedia.org/wiki/Standard_German#Pluricentricity)

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
Besides a grammatical suggestion, there's a minor nitpick with one word. I will let you decide on this word.

Regarding the discussion for standard german, I feel like "German Standard German" is the most fitting version for the wiki. Though, don't pin me on this. I'm open for further opinions.

It's pretty much useless trying to more accurately define *which* German should be used on the wiki I believe, as even linguistic academics aren't sure of what can be classified as German.

File: wiki/Beatmap/Category/de.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Zu Beginn hatten Approved Beatmaps keinen Einfluss auf das Score-Ranking eines Spielers. Dies änderte sich jedoch kurz nach dem Wechsel von Score-Ranking zu [Performance-Punkten](/wiki/Performance_points). Nach dieser Änderung wurde die Approved-Kategorie ausschließlich für Beatmaps mit mehr als 6 Minuten [Drain-Zeit](/wiki/Gameplay/Drain_time) verwendet.
+
+Im [Songauswahl-Bildschirm](/wiki/Client/Interface#songauswahl) werden Approved Beatmaps mit einem Häkchen-Symbol (![](/wiki/shared/status/approved.png)) markiert.
+
+### Einführung der Qualifiziert-Kategorie
+
+Die [Qualifiziert](#qualifiziert)-Kategorie wurde im August 2014 während der [Umstrukturierung des BATs](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat) eingeführt, um kürzlich gerankte Beatmaps eine größere Bühne zu geben. Wenn eine qualifizierte Beatmap disqualifiziert und zurück in die Pending-Kategorie verschoben werden musste, würde es die Statistiken der Spieler nicht beeinflussen, da die Kategorie von Anfang an für das Vergeben von temporären Scores bestimmt war. Dadurch wurden richtige Erwartungen gesetzt und das negative Feedback der Community gegenüber Unranks wurde drastisch reduziert.
```
[the remark this answers is not in the document]
Comment (Pasi4K5) [reply wiki/Beatmap/Category/de.md]: The suggestion is correct, I'm just an idiot. Anyway, I applied the suggestion.

**Comment by Pasi4K5** [comment]
> Besides a grammatical suggestion, there's a minor nitpick with one word. I will let you decide on this word.

Applied everything as suggested.

> Regarding the discussion for standard german, I feel like "German Standard German" is the most fitting version for the wiki. Though, don't pin me on this. I'm open for further opinions.

I definitely agree with you, German Standard German is the most widely used variety of German so it would fit the best IMO.

> It's pretty much useless trying to more accurately define _which_ German should be used on the wiki I believe, as even linguistic academics aren't sure of what can be classified as German.

Is it useless though? Yes, defining languages is difficult since the boundaries are pretty blurry at times, but there _are_ clearly defined varieties of Standard German, so don't you think that explicitly defining which one should be used might be a good idea?

**Comment by The-Last-Cookie** [comment]
Well, how about if we change

> Alle deutschen Artikel müssen Standarddeutsch verwenden.

to

> Alle deutschen Artikel müssen deutsches Standarddeutsch verwenden.

?

Feel free to open a PR!

**Comment by TicClick** [comment]
is there any remaining work, or is this ready to be merged?

**Comment by Pasi4K5** [comment]
I think @The-Last-Cookie and @FamousPig pointed out all issues they found. From my side it's ready for merging.

**Review by The-Last-Cookie — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]
